lib/ex_doc/formatter/epub/templates.ex

```diff
@@ -68,4 +70,16 @@ defmodule ExDoc.Formatter.EPUB.Templates do
   EEx.function_from_file(:defp, :head_template,
                          Path.expand("templates/head_template.eex", __DIR__),
                          [:config, :page])
+
+  @media_types
```

Comment:
I thought about it, but, in this case, the EPUB documents only support a mime-types subset, more information about this in [EPUB 3 Core Media Types](https://idpf.github.io/epub-cmt/v3/)